data/tlg0732/tlg011/tlg0732.tlg011.1st1K-grc1.xml

```diff
@@ -69,7 +68,7 @@
                             <title>Supplementum Aristotelicum</title>
                             <biblScope unit="volume">2.1</biblScope>
                         </series>
-                        <ref target="http://www.archive.org/stream/p1supplementumar02akaduoft">Internet Archive</ref>
+                        <ref target="https://archive.org/details/p1supplementumar02akaduoft/page/n123/mode/2up?view=theater">Internet Archive</ref>
```

Comment:
We don't generally use the view=theater attribute, I think.

Reply:
I will change that one file!